src/core/stdc/stdio.d

```diff
@@ -1689,6 +1739,28 @@ else version (CRuntime_UClibc)
     ///
     int  vsnprintf(scope char* s, size_t n, scope const char* format, va_list arg);
 }
+else version (WebAssembly)
+{
+    // No unsafe pointer manipulation.
+    @trusted
+    {
+        ///
+        void rewind(FILE* stream);
+        ///
+        pure void clearerr(FILE* stream);
```

Comment:
I think placing `pure` at the end of the function declaration is more standard idiomatic-D.